frontend/src/main.js

```diff
@@ -8,6 +8,7 @@ import LoadingSpinner from '@/components/LoadingSpinner'
 import AppLogo from '@/components/AppLogo'
 
 Vue.config.productionTip = false
+Vue.config.devtools = true
```

Comment:
?

Reply:
Vue browser extension doesn't work for production builds unless this is on

Reply:
I do npm run serve and npm start in each folder so it's always on dev branch.